### PR TITLE
Example test with config

### PR DESCRIPTION
**Pull request by katlapinka**
This change is [Reviewable](https://reviewable.io/reviews/open-cas/test-framework/5)

### CODE REVIEW

File: test_package/conftest.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+import os
+import sys
+sys.path.append(os.path.abspath('../psg_cas_l-opencas_val'))
```
Comment (Deixx) [inline test_package/conftest.py]: I'd rather have all the private information (like path to wrapper repo) in a gitignored config/configuration.py file with entry:
test_wrapper_dir = 'some_path'

and in this file:
import config.configuration as c
sys.path.append(os.path.abspath(c.test_wrapper_dir))

File: test_package/conftest.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+import os
+import sys
+import config.configuration as c
```
Comment (Deixx) [inline test_package/conftest.py]: add sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir)) before importing config
and add comment that user should provide config/configuration.py file

File: test_package/conftest.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+sys.path.append(os.path.abspath(c.test_wrapper_dir))
```
Comment (Deixx) [inline test_package/conftest.py]: add checks for c.test_wrapper_dir value and existence
add comment that user can enhance execution with own implementation of test_wrapper in provided path

File: test_package/conftest.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import test_wrapper
+
+
+@pytest.fixture()
+def prepare_and_cleanup(request):
+    """
+    This fixture returns the dictionary, which contains DUT ip, IPMI, spider, list of disks
+    """
+    yield from test_wrapper.run_test_wrapper(request)
```
Comment (Deixx) [inline test_package/conftest.py]: add simple alternative for the case when test_wrapper is not imported (preferably read dut info from a config file and add example file with described values required for execution - I think that ip and disks are enough)

File: test_package/example_test.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+LOGGER = logging.getLogger(__name__)
+
+
+def setup_module():
+    LOGGER.warning("Entering setup method")
+
+
+@pytest.mark.parametrize('prepare_and_cleanup', [{"disk_type": "optane", "count": 2}], indirect=True)
+def test_create_partition(prepare_and_cleanup):
```
Comment (Deixx) [inline test_package/example_test.py]: rename test

File: test_package/conftest.py (hunk 2 above)
Comment (katlapinka) [reply test_package/conftest.py]: done

File: test_package/conftest.py (hunk 3 above)
Comment (katlapinka) [reply test_package/conftest.py]: done

File: test_package/conftest.py (hunk 4 above)
Comment (katlapinka) [reply test_package/conftest.py]: done

File: test_package/example_test.py (hunk 5 above)
Comment (katlapinka) [reply test_package/example_test.py]: done

File: .gitignore [hunk 6]
```diff
@@ -201,3 +201,4 @@ dmypy.json
 .venv/
 
 # End of https://www.gitignore.io/api/python,pycharm
+/config/configuration.py
```
[the remark this answers is not in the document]
Comment (katlapinka) [reply .gitignore]: done

File: config/configuration.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
+#
```
[the remark this answers is not in the document]
Comment (katlapinka) [reply config/configuration.py]: done

**Pull request merged**